src/Objs/Homepage/HomepageEditingConfig.ts

```diff
@@ -26,6 +26,9 @@ provideEditingConfig(Homepage, {
     sitePortalOnlyMode: { title: 'Use portal-only mode?' },
     siteCartPage: { title: 'Location of cart page' },
     sitePortalOverviewPage: { title: 'Location of portal overview page' },
+    sitePortalOverviewFallbackPage: {
+      title: 'Location of portal overview page (redirect fallback)',
```

Comment:
* [ ] FYI A better explanation would give some details about the nature of the fallback, e.g., "Portal overview page for logged-out visitors".
* [ ] FYI How about we save editors from the hassle by making the "active" state redirect aware? I.e. the actual path of `sitePortalOverviewPage` follows the redirect. Such a helpers is a good idea in general b/c it can also be used as the preferred link target in navigations, saving roundtrips and helping crawlers of prerendered content.